app/js/handlebarsUserTable.js

```diff
@@ -11,18 +11,103 @@ function fillUserTable(HBTemplate) {
             return result.json()
         })
         .then(function(result) {
-            let user_list = document.querySelector(".user_list")
-            user_list.innerHTML = ""
-
-            if (result.success) {
-                result.data.forEach(function(userData) {
-                    let html = template(userData)
-                    user_list.innerHTML += html
-                })
-            } else {
-                user_list.innerHTML = "Please contact Admin, user list unavailable"
+            updateDisplayedUsers(result, template)
+        })
+}
+
+/**
+ *this fills the handlebars template with users that have not been soft deleted
+ *
+ * @param apiResponse
+ * @param template
+ */
+function updateDisplayedUsers(apiResponse, template) {
+    let user_list = document.querySelector(".user_list")
+    user_list.innerHTML = ""
+    if (apiResponse.success) {
+        apiResponse.data.forEach(function(userData) {
+            if (userData.deleted === "0") {
+                let html = template(userData)
+                user_list.innerHTML += html
             }
         })
+        addEditEventListeners()
+        addDeleteEventListeners()
+    } else {
+        user_list.innerHTML = "Please contact Admin, user list unavailable"
+    }
+}
+
+/**
+ * fills handlebars template by passing in object and inserts into the score_list div
+ *
+ * @param HBTemplate the handlebars template
+ * @param ObjFunction the function that creates an object of all fields required in scores page
+ */
+function fillEditModal(HBTemplate, userInfo) {
+    let template = Handlebars.compile(HBTemplate)
+    let modal_content = document.querySelector("#modal-content")
+
+    modal_content.innerHTML = ""
+
+    if (userInfo.name && userInfo.email && userInfo.id) {
```

Comment:
should userInfo.email be blue like name and id?

Reply:
This is jsut because name and id are recognised in any context by PHPStorm.